blackbox/test_jmx.py

```diff
@@ -230,26 +213,3 @@ def assert_valid_circuit_breaker_jmx_output(self, cb_name, output):
         self.assertRegex(output, r'trippedCount:\s+(\d+)')
         self.assertRegex(output, r'used:\s+(\d+)')
 
-
-class NoEnterpriseJmxIntegrationTest(unittest.TestCase):
```

Comment:
Should we re-add this as part of the ce test?

Reply:
I've added a new test case/file: `test_ce_no_enterprise.py`